allowlist

```diff
@@ -1,6 +1,8 @@
 telemetry/core
 org-mozilla-fenix/.*
 org-mozilla-reference-browser/.*
+org-mozilla-samples-glean/.*
+org-mozilla-tv-firefox/.*
 activity-stream/.*
```

Comment:
It is my understanding that `glean/.*` is not intended to be added to the allowlist, is that correct?

Reply:
Right. Glean is a library that sends no telemetry on its own.